app/src/main/java/com/xinto/opencord/ui/screens/mentions/MentionsViewModel.kt

```diff
@@ -57,14 +59,17 @@ class MentionsViewModel(
     init {
         initPager()
 
-        if (persistentGuildId > 0) {
-            viewModelScope.launch {
-                val guild = guilds.fetchGuild(persistentGuildId)
-                    ?: return@launch
+        persistentDataStore.observeCurrentGuild()
```

Comment:
this does not need to be updated when the current guild changes (it won't except for a tiny edge case)

it should stay the same for as long as the screen is open 

should probably add some getGuildId() or something to persistent store